x/src/test.rs

```diff
@@ -17,16 +23,66 @@ pub struct Args {
     #[structopt(long, short)]
     /// Only run unit tests
     unit: bool,
+
+    #[structopt(long)]
+    /// Do not fast fail the run if tests (or test executables) fail.
+    no_fail_fast: bool,
+    #[structopt(long)]
+    /// Do not run tests, only compile the test executables.
```

Comment:
no period at the end

```suggestion
    /// Do not run tests, only compile the test executables
```

Reply:
didn't see there was a commit suggestion button here?   nice feature - I'll be sure to use it in the future.

Reply:
done.